example.lua

```diff
@@ -64,3 +81,17 @@ minetest.register_chatcommand("sfs_lc", {
 		smartfs.create("asdinas",function() end)
 	end
 })
+
+-- attach form "s" to the demoblock node
+minetest.register_node("smartfs:demoblock", {
+	description = "SmartFS Demo block",
+	groups = {cracky = 3},
+	tiles = {"demo.png"},
+	on_receive_fields = smartfs.nodemeta_on_receive_fields
+})
+
+minetest.register_on_placenode(function(pos, newnode, placer, oldnode, itemstack, pointed_thing)
```

Comment:
It is an example, I prefer the on_placenode for nodes placed by player and on_construct for generated nodes.